Add documentation for {block,allow}_new_chunks (#237)
timescale/docs.timescale.com-content: api.md

>:WARNING: The management of tablespaces on hypertables is currently an
experimental feature.

---
## block_new_chunks() [](block_new_chunks)

Prevent new chunks from being created on a data node without losing
access to any of the existing data. Note than any chunks already
existing on the data node may continue to receive further INSERTs and
UPDATEs.

This can be useful if one data node is running low on space or simply

> as a tool to fine tune how data is distributed as more distributed

Reviewer: ```suggestion
as a tool to fine-tune how data is distributed as more distributed
```

> hypertables are created. To resume using a data node once blocked for

Author: I didn't see a good way of adding this without being exceptionally verbose.  I did change the first sentence to highlight this as a tool for safe destaging though.

a hypertable, use [`allow_new_chunks`](#allow_new_chunks).

#### Required Arguments [](block_new_chunks-required-arguments)

| Name             | Description                       |
|------------------|-----------------------------------|
| `data_node_name` | Name of data node to stop creating chunks for the distributed hypertable |

#### Optional Arguments [](block_new_chunks-optional-arguments)

| Name         | Description                            |
|--------------|----------------------------------------|

> | `hypertable` | Name of the distributed hypertable for which chunks creation is being stopped. If NULL, no new chunks will be created for any hypertables. |

Reviewer: ```suggestion
| `hypertable` | Name of the distributed hypertable for which chunk creation is being blocked. If NULL, no new chunks will be created for any hypertables. |
```

| `force`      | Force blocking of the data node even if that means that the replication factor will no longer be reachable for new chunks. Note that it will never be allowed to block chunks on the last data node for a distributed hypertable.         |

#### Errors

Blocking chunks on a data node is not permitted:
- If it would result in no available data nodes for any future chunks
on a hypertable
- If it would result in future chunks being under-replicated for the
distributed hypertable (without the `force` argument)

> >:TIP: Replication is currently experimental, and not a supported feature

Reviewer 2: ```suggestion
>:TIP: Replication is currently experimental, and not a supported feature.
```
